src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs

```diff
@@ -56,5 +56,23 @@ internal static class HttpContextExtensions
             // e.g. "GET /pets/1"
             return $"{method} {route}";
         }
+
+        public static SentryTraceHeader? TryGetSentryTraceHeader(this HttpContext context)
+        {
+            try
+            {
+                if (!context.Request.Headers.TryGetValue(SentryTraceHeader.HttpHeaderName, out var value))
+                {
+                    return null;
+                }
+
+                return SentryTraceHeader.Parse(value.ToString());
+            }
+            catch
+            {
+                // Parsing can fail
+                return null;
```

Comment:
No logger.logerror? We could be hiding a bug here

Reply:
@bruno-garcia updated